shared/engine/index.js

```diff
@@ -281,4 +279,12 @@ export type ResponseType = {
   error: (...args: Array<any>) => void,
 }
 
-export default engine
+let engine
+const makeEngine = () => {
```

Comment:
Alternatively a more explicit `setupEngine` called in app startup would be a welcome addition.